.github/workflows/build_stable.yml

```diff
@@ -50,6 +51,7 @@ jobs:
       repo: erpnext
       version: "13"
       push: ${{ github.repository == 'frappe/frappe_docker' && github.event_name != 'pull_request' }}
+      platforms: linux/amd64
```

Comment:
Set `linux/amd64,linux/arm64`. Otherwise, what is the point of this PR? 😃

Reply:
done